src/main/kotlin/uk/gov/dluhc/printapi/service/PrintResponseFileReadinessService.kt

```diff
@@ -0,0 +1,47 @@
+package uk.gov.dluhc.printapi.service
+
+import mu.KotlinLogging
+import org.apache.commons.lang3.time.StopWatch
+import org.springframework.stereotype.Service
+import uk.gov.dluhc.printapi.config.SftpProperties
+import uk.gov.dluhc.printapi.messaging.models.ProcessPrintResponseFileMessage
+
+private val logger = KotlinLogging.logger {}
+
+@Service
+class PrintResponseFileReadinessService(
+    private val sftpService: SftpService,
+    private val sftpProperties: SftpProperties,
+    private val printMessagingService: PrintMessagingService
+) {
+
+    fun markPrintResponseFileForProcessing() {
```

Comment:
Perhaps `markAndSubmitPrintResponseFileForProcessing`()? I think some kdoc, perhaps at the class level, wouldn't hurt btw.